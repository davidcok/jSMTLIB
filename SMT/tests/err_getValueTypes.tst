; check types in the get-value commands
(set-option :produce-models true)
(set-logic QF_UF)
(declare-sort A 0)
(declare-fun x () A)
(declare-fun y () Bool)
(assert true)
(check-sat)
(get-value (x y (and x y) ))
