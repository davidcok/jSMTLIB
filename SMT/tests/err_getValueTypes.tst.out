unsupported
success
success
success
success
success
sat
(get-value (x y (and x y) ))
                ^^^^^^^^^
(error "Unknown predicate symbol and with argument types A Bool")
